posts/git-cherry-pick/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Cherry-Pick on GitHub"
+author: "Robyn Marowitz"
+date: "2023-09-26"
+description: "Committing a mistake in GitHub can feel so scary, cherry picking is a tool to help you fix your history when something like that happens. This post lays out simple and clear instructions on the cherry picking method."
+categories: [github, open-science]
+image: "cherry_pick.jpeg"
+---
+
+## What is git cherry-pick?
+
+It is the act of picking a commit from one branch and applying it to another branch.  
+
+## When should I use cherry-pick?
```

Comment:
```suggestion
## When should I cherry-pick?
```

If we say "use" I feel we should reference the command i.e. "When should I use `git cherry-pick`?"

What do you think? Just a gut preference, introspected too deep in to why :)